install_rock.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-if [ "$(uname -o)" == "Msys" ]; then
-	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*zip\>')"
-	wget https://github.com/cogneco/rock/releases/$tmp
-	echo ${tmp##*/}
-	7z e ${tmp##*/}
-else
```

Comment:
> And so I did, a long time ago :)

Oh, right, so you did. I was expecting some news below this, but I realize that was incorrect of me.